Bump driver version in order to distinguish DUP driver from in-tree one.

Index: src/drivers/net/ethernet/intel/i40evf/i40evf_main.c
===================================================================
--- src.orig/drivers/net/ethernet/intel/i40evf/i40evf_main.c	2017-04-27 20:23:02.995045808 +0200
+++ src/drivers/net/ethernet/intel/i40evf/i40evf_main.c	2017-04-27 20:23:38.578624032 +0200
@@ -34,7 +34,7 @@
 static const char i40evf_driver_string[] =
 	"Intel(R) 40-10 Gigabit Virtual Function Network Driver";
 
-#define DRV_KERN "-k"
+#define DRV_KERN "-k_dup7.3"
 
 #define DRV_VERSION_MAJOR 1
 #define DRV_VERSION_MINOR 6
